src/windows/windows.js

```diff
@@ -301,7 +301,7 @@ define(['jquery', 'lodash', 'navigation/navigation', 'windows/tracker', 'jquery.
 
             var options = {
                 showOn: 'both',
-                numberOfMonths: 2,
```

Comment:
@arnabk the issue was that the binary-style is not supplying a css-style for the other month, this caused the date picker to look funny. I have checked, changing the month to 1 won't cause any issues for webtrader.